### PR TITLE
Header feature

### PR DESCRIPTION
**Pull request by RaviTejaM9602**
This is a duplicate of Figma mobile template one, header section

### CODE REVIEW

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @RaviTejaM9602 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub flow :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ]  The Repository should have a descriptive Pull Request message and a Pull Request meaningful title, this is important because we let the reviewer or any other user the intent of the changes we are about to merge with a production branch.
- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RaviTejaM9602** [comment]
Pull request
Ravi Teja M
What does this PR do?
This PR is the continuation of the skeleton of a portfolio page. (mobile version)
Update Logo and Nav Bar as per above review comments 
Added the section, ul, li, tags in the HTML file
Used CSS file to style up the tags in HTML file
created test.MD files that have been ignored
How should it be tested
clone the repository
git clone https://github.com/RaviTejaM9602/Portfolio.git

File: index.html [hunk 1]
```diff
@@ -4,9 +4,47 @@
...
 </head>
 <body>
-    <h1>Designing Portfolio Skeleton</h1>
+  <!--Header Section -->
+
+  <main class="head-section"> 
+      <nav class="top d-flex">
+        <h1>My Logo</h1>
+        <i class="fas fa-bars"></i>
+      </nav>
```
Comment (andersonlebon) [inline index.html:20]: - [ ] Good job on implementing the previous request. However, you still have something to improve:
- Kindly put this `<nav>` tag inside a `<header>` tag.

File: index.html [hunk 2]
```diff
@@ -4,9 +4,47 @@
...
+  <main class="head-section"> 
+      <nav class="top d-flex">
+        <h1>My Logo</h1>
+        <i class="fas fa-bars"></i>
+      </nav>     
+
+     <!--Main Section -->
+
+     <section  class="main-section">
+      <h2 class="intro">I'm Ravi Teja<span>glad to see</span><span>
```
Comment (andersonlebon) [inline index.html:25]: - [ ] As I sad before the the main title(your name) should be a` <h1>` and not a `<h2>` tag. use this
```
<h1 class="intro">I'm Ravi Teja<span>glad to see</span><span>
 you!!</span></h1>
```

File: index.html [hunk 3]
```diff
@@ -4,9 +4,47 @@
...
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
+            <li><a href="#"><img src="images/Vector.svg" alt="Vector" ></a></li>         
+            <li><a href="#"><i class="fab fa-github"></i></a></li>            
+            <li><a href="#"><img src="images/Finger.svg" alt="Finger"></a></li>
+          </ul>     
+      </section>  
+   </section>  
+   
+   </main>
```
Comment (andersonlebon) [inline index.html:47]: - [ ] GOOD JOB :clap: :clap: I can see you have improved your styling you are almost there. you just have only to set the `font-family` as well as the `font-size`. Kindly follow the template style

| your design | the template|
| --------------- | ---------------- |
Screenshot from 2021-11-26 04-38-02 | 143390039-f6c2a839-f2f9-4217-acae-05cf29dcefb2

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @RaviTejaM9602 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RaviTejaM9602** [comment]
I have made required changes as mentioned above please find the screen shot for reference.

portfolio

**Comment by andersonlebon** [comment]
- [ ] OOh CONGRATULATIONS :clap:  i can see you improved your design kindly change `Marian` to your name and you will be ok

**Comment by RaviTejaM9602** [comment]
Done with the suggested Change 
Capture

**Review by andersonlebon — APPROVED** [review]
Hi @RaviTejaM9602 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 
- Your design is now looking good :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**